lang/cs/Org.Apache.REEF.Client/YARN/RESTClient/IYarnRMClient.cs

```diff
@@ -37,6 +37,22 @@ internal interface IYarnRMClient
 
         Task<Application> GetApplicationAsync(string appId, CancellationToken cancellationToken);
 
+        /// <summary>
+        /// This API returns information about all the applications maintained
+        /// by YARN RM in the cluster by invoking REST API.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
Empty xmldoc tag - remove it, or, better yet, populate with some meaningful docstring.